slacker/utility.py

```diff
@@ -72,3 +74,26 @@ def parse_args():
 
   args = parser.parse_args(args)
   return (args, cmd_args)
+
+def verify_token(token):
```

Comment:
I wonder if we should convert this to be a decorator that we apply to `action` in the command, at least where the commands require a valid auth token? Although that might be a bit spamming if we are making an auth test call before running the command.

Reply:
Hm that would make too many requests to the API that are unnecessary. We will get an unauthed error if using an invalid token anyway.